Src/PowerShell/load-used-actions.ps1

```diff
@@ -46,9 +46,11 @@ function  GetActionsFromWorkflow {
                         if ($null -ne $uses) {
                             Write-Host "   Found action used: [$uses]"
                             $actionLink = $uses.Split("@")[0]
+                            $actionRef = $uses.Split("@")[1]
```

Comment:
Little tidbit, let's only split it once and then use the parts
```suggestion
                            $splitted = $uses.Split("@")
                            $actionLink = $splitted[0]
                            $actionRef = $splitted[1]
```

Reply:
Indeed, thx for the assistance (first time I develop in Powershell :) )